### PR TITLE
Connect to upower-observe, needed for the battery indicator

### PR DESCRIPTION
**Pull request by kenvandine**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ cp /snap/ubuntu-desktop-session/current/ubuntu-desktop-session.desktop /usr/shar
 for plug in hardware-observe home login-session-observe login-session-control \
             mount-observe network-control network-observe polkit-agent \
             process-control system-observe shutdown shell-config-files \
-            snapd-control; do
+            upower-observe snapd-control; do
```
Comment (robert-ancell) [inline setup.sh:16]: This should be in alphabetical order.

**Comment by robert-ancell** [comment]
On a fresh install this already seems to be connected, are you not seeing that case?

**Comment by kenvandine** [comment]
> 
I had to connect it myself

**Comment by robert-ancell** [comment]
James also has this connected by default. However when we look at the snapd configuration it appears this should not be autoconnected. So this PR is specifying the right behaviour, even though it is working for some of us right now without it.

**Review by robert-ancell — APPROVED** [review]

**Pull request merged**